### ISSUE TITLE
`run_fcst` fails on Hera

### ISSUE THREAD

**Issue by danielabdi-noaa** [ufs-community/ufs-srweather-app#711]
## Expected behavior

`run_fcst` should run successfully on Hera.

## Current behavior

`run_fcst` on the `custom_GFDLgrid` test case fails on Hera. I haven't investigated this thoroughly so the issue could
be on my side. I did checkout and rebuilt all submodules from scratch though, and it still failed.

## Machines affected

Hera

## Steps To Reproduce

Run the `custom_GFDLgrid` test case on Hera.

## Additional Information (optional)

```
 448  After adi: W max =    2.930210      min =   -1.478376
 449  na_ini Z500   5809.308       5790.431      0.0000000E+00   5880.680
 450 forrtl: severe (19): invalid reference to variable in NAMELIST input, unit -5, file Internal Formatted NML Read, line -1, position 22
 451 Image              PC                Routine            Line        Source
 452 ufs_model          0000000004327E78  Unknown               Unknown  Unknown
 453 ufs_model          0000000004363F87  Unknown               Unknown  Unknown
 454 ufs_model          0000000004361F60  Unknown               Unknown  Unknown
 455 ufs_model          00000000030B4947  gfs_typedefs_mp_c        3621  GFS_typedefs.F90
 456 ufs_model          000000000302864F  gfs_init_mp_gfs_i          68  GFS_init.F90
 457 ufs_model          0000000001C78F79  atmos_model_mod_m         690  atmos_model.F90
 458 ufs_model          0000000001B30603  module_fcst_grid_         764  module_fcst_grid_comp.F90
 459 ufs_model          00000000009B4484  _ZN5ESMCI6FTable1        2167  ESMCI_FTable.C
 460 ufs_model          00000000009B8406  ESMCI_FTableCallE         824  ESMCI_FTable.C
 461 ufs_model          000000000111C23F  _ZN5ESMCI3VMK5ent        2308  ESMCI_VMKernel.C
 462 ufs_model          00000000010AAD0A  _ZN5ESMCI2VM5ente        1216  ESMCI_VM.C
 463 ufs_model          00000000009B5B17  c_esmc_ftablecall         981  ESMCI_FTable.C
 464 ufs_model          00000000008C7DBD  esmf_compmod_mp_e        1222  ESMF_Comp.F90
 465 ufs_model          0000000000BB3C36  esmf_gridcompmod_        1407  ESMF_GridComp.F90
 466 ufs_model          0000000001B2185E  fv3gfs_cap_mod_mp         383  fv3_cap.F90
 467 ufs_model          0000000000EF541F  _ZNK5ESMCI13Metho         377  ESMCI_MethodTable.C
 468 ufs_model          0000000000EF53A2  _ZN5ESMCI11Method         563  ESMCI_MethodTable.C
 469 ufs_model          0000000000EF5A00  c_esmc_methodtabl         347  ESMCI_MethodTable.C
 470 ufs_model          00000000008F335B  esmf_attachmethod        1280  ESMF_AttachMethods.F90
 471 ufs_model          00000000023EE959  nuopc_modelbase_m         695  NUOPC_ModelBase.F90
 472 ufs_model          00000000009B4484  _ZN5ESMCI6FTable1        2167  ESMCI_FTable.C
 473 ufs_model          00000000009B8406  ESMCI_FTableCallE         824  ESMCI_FTable.C
 474 ufs_model          000000000111C23F  _ZN5ESMCI3VMK5ent        2308  ESMCI_VMKernel.C

```

**Comment by christinaholtNOAA** [comment]
@danielabdi-noaa This error typically crops up when there is a mismatch between the compiled version of the code and the version of the SRW being used. Did you run manage_externals and re-build the code after the latest update to the model hash?

**Comment by danielabdi-noaa** [comment]
@christinaholtNOAA Yes. I removed all sub-folders under src/ manually, checked out all packages from develop branch and re-built all binaries, checked that all binaries under exec/ have today's date etc. However, I am still not sure if it is not my fault. I had a similar issue testing AQM in the past where I swear I did everything right until it turned out I didn't. 
https://github.com/ufs-community/ufs-srweather-app/pull/682#issuecomment-1482861055
If someone can run customGFDL grid successfully on Hera, I will close this issue.
